### PR TITLE
Node 18

### PR DESCRIPTION
**Pull request by tomrf1**
The node update caused `npm install` to fail because of some incompatible peer dependencies that were previously not flagged as a problem. Setting overrides 'fixes' this for now.
It was also necessary to upgrade webpack

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +53,11 @@
   "resolutions": {
     "csstype": "^2.6.18"
   },
+  "overrides": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```
Comment (tomrf1) [inline package.json:58]: these react overrides are for the benefit of `react-beautiful-dnd` (a drag-and-drop library). This is no longer actively maintained, so we should consider removing/replacing it

File: package.json [hunk 2]
```diff
@@ -52,6 +53,11 @@
...
+    "prettier": "^1.19.1"
```
Comment (tomrf1) [inline package.json:59]: This isn't necessary if we update our `prettier` dependency, but newer versions want to reformat the entire codebase so I didn't want to do this now

File: package.json [hunk 3]
```diff
@@ -4,9 +4,10 @@
     "@types/material-ui": "^0.21.7",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.0.1",
+    "@types/react": "^17.0.58",
```
Comment (tjmw) [inline package.json:7]: Should this be `"~17.0.58"`? I think these types libraries guarantee compatibility within the minor version?

File: package.json (hunk 1 above)
Comment (tjmw) [reply package.json:58]: Earlier on when we were looking at this I was translating `dnd` => dungeons and dragons 😄

**Review by tjmw — APPROVED** [review]
Nice!

**Pull request merged**